### DISCUSSION TITLE
i3 4.21 released!

### DISCUSSION THREAD

**Discussion by stapelberg** [i3/i3#5146] [Announcements]
Release notes can be found at https://i3wm.org/downloads/RELEASE-NOTES-4.21.txt

**Comment by orestisfl** [comment]
Hmm, the [userguide](https://i3wm.org/docs/userguide.html) still doesn't seem to mention tiling drag for me even though I see "Documentation for i3 v4.21" in https://i3wm.org/docs/

**Reply by stapelberg** [reply]
https://i3wm.org/docs/userguide.html#_moving_tiling_containers_with_the_mouse shows up correctly for me? 
![userguide](https://user-images.githubusercontent.com/55506/191567539-edff3611-2f59-4d82-9a9c-3924fafb5802.jpg)

<s>Maybe try shift+reload in your browser, it might still have the page cached?</s> (Nevermind: if it shows v4.21, it can’t be cached.)

**Reply by orestisfl** [reply]
Yes, correct. Never mind :facepalm:.

**Comment by UnclassedPenguin** [comment]
I <3 i3 so much, thanks :)

**Comment by moson-mo** [comment]
Just noticed the date in the release notes is `2021-10-19`.
Probably a leftover from the previous 4.20 release 😉

**Reply by stapelberg** [reply]
Yes, thanks. We won’t fix that, as the date that `i3 --version` reports is correct, and it’s not worth doing another release just for the date.

**Reply by moson-mo** [reply]
Oh, I see. That file is also part of the git repo.
I thought it's just some file on the webserver 😄

**Reply by orestisfl** [reply]
Can we not just update this file https://github.com/i3/i3.github.io/blob/main/downloads/RELEASE-NOTES-4.21.txt?

**Reply by stapelberg** [reply]
We could, but then it diverges from what’s in the release tarball, and that’s not great either. I’d just leave it as-is, especially because we’ll do a bugfix release soon anyway :)

**Comment by noodlor** [comment]
I have been following this feature for years; so glad it finally happened!

I'm seeing some unexpected behavior: when there's a single window in a workspace and I drag it onto itself (drag the titlebar to the center of the window), it moves to the scratchpad.

**Reply by orestisfl** [reply]
Please open a new issue with reproduction instructions. You can skip the log for now.

**Comment by erebion** [comment]
What setting do I have to set to revert mouse tiling to the previous behaviour?

**Reply by erebion** [reply]
Okay, finally found out that the new feature is callec "tiling drag" and can be turned off with `tiling_drag off`.

I'd like to see something like this (name of the exact setting) in the release notes in the future so it's easier to find. :)

**Reply by stapelberg** [reply]
It is in the release notes, but for 4.21.1: https://i3wm.org/downloads/RELEASE-NOTES-4.21.1.txt, which is when the tiling drag option was introduced.

Yes, ideally it should have been a part of 4.21 to begin with, but hindsight is 20/20.
